### PR TITLE
port of prisma check OCI IAM password policy for local (non-federated) users does not have minimum 14 characters 

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
Addresses #1969

Looks to see that the policy is set to 14 or more.

### CODE REVIEW

File: checkov/terraform/checks/resource/oci/IAMPasswordLength.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class IAMPasswordLength(BaseResourceCheck):
+    def __init__(self):
+        name = "OCI IAM password policy for local (non-federated) users has a minimum 14 characters"
+        id = "CKV_OCI_18"
+        supported_resources = ['oci_identity_authentication_policy']
+        categories = [CheckCategories.GENERAL_SECURITY]
```
Comment (gruebel) [inline checkov/terraform/checks/resource/oci/IAMPasswordLength.py:10]:
```suggestion
        categories = [CheckCategories.IAM]
```

File: checkov/terraform/checks/resource/oci/IAMPasswordLength.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class IAMPasswordLength(BaseResourceCheck):
+    def __init__(self):
+        name = "OCI IAM password policy for local (non-federated) users has a minimum 14 characters"
```
Comment (gruebel) [inline checkov/terraform/checks/resource/oci/IAMPasswordLength.py:7]:
```suggestion
        name = "OCI IAM password policy for local (non-federated) users has a minimum length of 14 characters"
```

File: checkov/terraform/checks/resource/oci/IAMPasswordLength.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    def scan_resource_conf(self, conf):
+        if 'password_policy' in conf.keys():
+            rules = conf.get("password_policy")[0]
+            if 'minimum_password_length' in rules:
+                passwordlength=rules.get("minimum_password_length")
+                if passwordlength[0] < 14:
+                    return CheckResult.FAILED
+                return CheckResult.PASSED
+            return CheckResult.FAILED
+        return CheckResult.FAILED
```
Comment (gruebel) [inline checkov/terraform/checks/resource/oci/IAMPasswordLength.py:22]: would be great, if you can add the evaluated keys like here
https://github.com/bridgecrewio/checkov/blob/master/checkov/terraform/checks/resource/aws/RedShiftSSL.py

**Review by gruebel — COMMENTED** [review]
some notes 🙂

File: checkov/terraform/checks/resource/oci/IAMPasswordLength.py (hunk 3 above)
Comment (JamesWoolfenden) [reply checkov/terraform/checks/resource/oci/IAMPasswordLength.py:22]: what do they do?

**Review by gruebel — APPROVED** [review]
🍻

**Pull request merged**